packages/rainbow-sprinkles/src/assignClasses.ts

```diff
@@ -29,7 +26,7 @@ export function assignClasses<Conditions extends BaseConditions>(
     // Quietly warn
     // eslint-disable-next-line no-console
     console.error(
-      `Homebase: invalid value provided to prop '${propName}'. Expected one of ${Object.keys(
+      `Rainbow Sprinkles: invalid value provided to prop '${propName}'. Expected one of ${Object.keys(
```

Comment:
Heh noticed this today also, I'll remove my change in #25 👍

This will resolve #22 also 💯